### PR TITLE
inventory-fix

### PR DESCRIPTION
**Pull request by Psalmz777**
#### Summary of Changes (What does this PR do?)
Fix Inventory Dockerfile and Settings.py to start container

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
thanks @Psalmz777

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 1]
```diff
@@ -388,6 +383,8 @@ jobs:
             --port=8000 \
             --cpu=1000m \
             --memory=256Mi \
+            --update-secrets=/etc/env/.env=sta-env-netmanager:latest \
```
Comment (Baalmart) [inline .github/workflows/deploy-frontend-pr-previews.yml:386]: Hi @Psalmz777 , why am I seeing `netmanager` being mentioned in a PR for` inventory`?

**Review by Baalmart — CHANGES_REQUESTED** [review]
Hi @Psalmz777 , 

1. please add an appropriate PR description and remove any template notes from the PR's description
2. I have also left a small inquiry for your review

**Comment by Psalmz777** [comment]
> Hi @Psalmz777 ,
> 
> 1. please add an appropriate PR description and remove any template notes from the PR's description
> 2. I have also left a small inquiry for your review
@Baalmart  Fixed

**Review by Baalmart — APPROVED** [review]
thanks @Psalmz777

**Pull request merged**